array_methods_assignment.js

```diff
@@ -0,0 +1,519 @@
+// # Exercises
```

Comment:
Overall great job !! I see that your understanding the array methods. Really glad you pushed yourself to complete the bonus questions !